### ISSUE TITLE
Не пропускать поля для которых не задано правило валидации.

### ISSUE THREAD

**Issue by Prots** [koorchik/LIVR#17]
Не совсем понятно почему спецификация допускает наличие в исходном пакете полей, для которых не заданы правила валидации? И в резалтсете мы получаем обрезанный набор исходных данных вместо ошибки валидации.

**Comment by SDSWanderer** [comment]
Понятное дело что оставить эти поля как есть нельзя по причине возможных проблем с безопасностью. Бросать ошибку - нормальный вариант, но подойдет не всем. Тем кому это нужно  - довольно просто реализовать такое поведение дополнительной проверкой, а вот тем кому нет - не возможно было бы реализовать игнорирование пустых полей в случае если бы валидатор бросал ошибку. Также, нормальным решением как по мне является специальная (специфичная для конкретной реализации) не обязательная опция меняющая это поведение.

**Comment by koorchik** [comment]
В целом идея неплохая. Это может быть strict режим валидации, возможно добавить в будущих версиях спецификации.

**Comment by Prots** [comment]
Да, неплохо бы добавить в валидатор возможность передачи набора опций, одной из которых может быть "strict".
